babel.config.js

```diff
@@ -1,6 +1,8 @@
 module.exports = {
   presets: [
     ["@babel/preset-env", { targets: { node: "current" } }],
```

Comment:
```suggestion
    ["@babel/preset-env", { targets: { node: "10" } }],
```
since node 10 is the lowest supported version